### ISSUE TITLE
Random test failures (time-related?)

### ISSUE THREAD

**Issue by eserte** [gittex/Sys-RotateBackup#3]
See http://matrix.cpantesters.org/?dist=Sys-RotateBackup%200.12 for a pass/fail overview, which looks quite randomly. My impression is that the failures could be dependent on the time of day the tests ran.

**Comment by dominikschulz** [comment]
Yes they are, these test do not fully mock time. Need to do that some time ...
